service/src/main/java/bio/terra/externalcreds/controllers/OauthApiController.java

```diff
@@ -124,7 +124,7 @@ public ResponseEntity<Void> deleteLink(Provider provider) {
 
     // This can be deleted once there's no longer a dependency on Bond's Datastore
     if (ProviderUtils.isFenceProvider(provider)) {
-      fenceProviderService.deleteFenceLink(samUser.getSubjectId(), provider);
+      fenceProviderService.deleteBondFenceLink(samUser.getSubjectId(), provider);
```

Comment:
Why would we only delete the linked account from Bond? I thought it could be in either ECM or Bond once we make the Terra UI talk to ECM 🤔

Reply:
Deleting the linked account in ECM cascade deletes the Fence Account Key in its DB! So, we only need to delete the linked account in ECM. We just _also_ need to tell Bond that we're deleting the linked account

Reply:
Oh that makes sense!